Directory.Build.props

```diff
@@ -15,6 +15,8 @@
     <LangVersion>Latest</LangVersion>
     <!-- Generate snupkg package -->
     <IncludeSymbols>true</IncludeSymbols>
+    <NoWarn>1701;1702</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Note that we are also setting this to true in src\devices\Directory.Build.props which I know doesn't import this or doesn't get imported by S.D.G or by I.D.B but we could probably find a central place where it will apply for both, for example in Compilers.props

Reply:
Moved to Compilers.props and added a condition to disable CS1591 (documentation) for test assemblies.